kuksa-client/setup.cfg

```diff
@@ -27,7 +27,6 @@ install_requires =
     websockets >= 10.1
     cmd2 >= 1.4, <2.0
     pygments >= 2.5
-    importlib_metadata >= 3.6 ; python_version < "3.8"
```

Comment:
This line was useless as it only should be installed for Python versions below 3.8, but we require at least 3.8 so it will never trigger